test/controllers/registrations_controller_test.rb

```diff
@@ -13,8 +13,12 @@ class RegistrationsControllerTest < ActionController::TestCase
   test 'create registration' do
     skip unless ENABLE_PUBLIC_REGISTRATION
 
+    creation_classes = %w[
```

Comment:
En otros tests está como:
```suggestion
    counts = %w[
```

Solo para "seguir la tradición".